gdal/swig/python/fallback_build_mingw32_under_unix_py37.sh

```diff
@@ -22,7 +22,7 @@ LINKFLAGS="-L../../.libs -lgdal -L${PYTHONHOME}/libs -l${PYTHONLIB}"
 CFLAGS="-O2 -D__MSVCRT_VERSION__=0x0601"
 
 # Run native python
-wine "$PYTHONHOME/python" setup.py build
+wine "$PYTHONHOME/python3" setup.py build
```

Comment:
should be reverted as the cause of the failure for the mingw_w64 CI job. The windows Python 3.7 install probably only ships python.exe